cruise.umple/test/cruise/umple/compiler/UmpleClassTest.java

```diff
@@ -824,6 +824,28 @@ public void getCodeInjectionUnfoundAttributesMethodError_complex()
   }
 
   @Test
+  public void getCodeInjectionParametersIgnoredError()
+  {
+    boolean custom = false;
+    boolean generated = false;
+    String code = "class A{a; void foo(int k) { foo(k-1); } after custom foo(int) { return true; } after generated setA(int) { return true; }}";
+    try {
+      UmpleModel model = getModel(code);
+      model.run();
+      ParseResult result = model.getLastResult();
+      List<ErrorMessage> errors = result.getErrorMessages();
+
+      Assert.assertEquals(errors.size(), 2);
+    } catch (Exception e) {

```

Comment:
This should catch a lower exception type since you want to be specific. Otherwise, it'll catch everything and may pass unknowingly.